### PR TITLE
Update the Description of GraphQL Tool

### PR DESCRIPTION
**Pull request by DimuthuMadushan**

### CODE REVIEW

**Review by ThisaruGuruge — DISMISSED** [review]

File: components/learn/boxes/Boxes.js [hunk 1]
```diff
@@ -137,8 +137,7 @@ export default function Boxes(props) {
                           <a href={`${prefix}/learn/graphql-tool`} className={styles.titleLink}>
                             GraphQL tool
                           </a>
-                        </p>
-                        <p className={styles.description}>Generate GraphQL client skeletons in Ballerina.</p>
+                        <p className={styles.description}>Generate GraphQL service skeletons for a given GraphQL schema and generate schema for a given Ballerina GraphQL service.</p>
```
Comment (praneesha) [inline components/learn/boxes/Boxes.js:140]:
```suggestion
                        <p className={styles.description}>Generate the Ballerina GraphQL service skeleton for a given GraphQL schema and vice versa.</p>
```

Comment (DimuthuMadushan) [reply components/learn/boxes/Boxes.js:140]: IMO it's better to describe both operations here. BTW we are generating the schema from the service, not from the service skeletons. In that case, can we use `vice versa`?

Comment (praneesha) [reply components/learn/boxes/Boxes.js:140]: +1

Comment (DimuthuMadushan) [inline components/learn/boxes/Boxes.js:140]:
```suggestion
                        <p className={styles.description}>Generate GraphQL service skeleton for a given GraphQL schema and generate schema for a given Ballerina GraphQL service.</p>
```

**Review by praneesha — APPROVED** [review]

**Pull request merged**